src/Compilers/VisualBasic/Test/Semantic/Compilation/CompilationAPITests.vb

```diff
@@ -348,9 +348,10 @@ End Namespace
 
             Dim result = c.Emit(stream, options:=options)
 
+            Dim nameCannotStartWithWhitespace = New CodeAnalysisResourcesLocalizableErrorArgument(NameOf(CodeAnalysisResources.NameCannotStartWithWhitespace))
             result.Diagnostics.Verify(
                 Diagnostic(ERRID.ERR_InvalidDebugInformationFormat).WithArguments("-1"),
-                Diagnostic(ERRID.ERR_InvalidOutputName).WithArguments(CodeAnalysisResources.NameCannotStartWithWhitespace),
```

Comment:
>CodeAnalysisResources.NameCannotStartWithWhitespace [](start = 70, length = 51)

I think we should simply hardcode "Name cannot start with whitespace." string here.